insights/parsers/named_conf.py

```diff
@@ -16,31 +16,44 @@
 class NamedConf(NamedCheckconf):
     """
     Class for parsing the file ``/etc/named.conf```, We use class ``NamedCheckConf`` to parse most
-    of the named.conf configurations and class ``NamedConf`` to parse the `include` directives.
+    of the named.conf configurations and class ``NamedConf`` to parse the `include` directives and
+    the `allow-recursion` value from the `options` section.
 
     .. note::
         Please refer to the super-class :py:class:`insights.parsers.named_checkconf:NamedCheckConf`
         for more usage information.
 
     Attributes:
-        includes (list): List of files in 'include' section.
+        includes (list): List of files in 'include' and 'allow-recursion' value from the 'options' section.
 
     Raises:
         SkipException: When content is empty or cannot be parsed.
 
     Examples:
         >>> named_conf.includes
         ['/etc/crypto-policies/back-ends/bind.config']
+        >>> named_conf.allow_recursion_address
+        ['localnets', 'localhost', '192.168.10.1/24']
     """
 
     def parse_content(self, content):
         includes = []
+        allow_recursion_address = []
+        options_flag = False
         super(NamedConf, self).parse_content(content)
 
         try:
-            for line in [l for l in content if l.strip().startswith('include ') and ';' in l]:
-                includes.append(line.split(';')[0].replace('"', '').split()[1])
+            for line in content:
+                if line.strip().startswith('include ') and ';' in line:
```

Comment:
the `line.strip()` is used times inner the loop, better assign it to a local variable to reduce compute.

```line = line.strip()```

Reply:
Yes, it's a great point. Updated.